### PR TITLE
 Populate null columns in repo_info table

### LINKED ISSUE #2675: repo_info table not populating certain columns
The following columns are always returning null: 

Ones that I am looking to use:
- contributing_file 
- security_issue_file
- keywords (if that is referring to the topics, renaming to be discussed if so) 

Others not populated: 
- pull_requests_enabled 
- pages_enabled 
- issue_contributors_count
- changelog_file

### PR DESCRIPTION
**Pull request by guptapratykshh**
**Description**
added data collection for six null columns in [repo_info](cci:1://file:///Users/pratykshgupta/Desktop/pg-augur/augur/tasks/github/repo_info/core.py:199:0-350:54) table -`contributing_file`, `security_issue_file`, `changelog_file`, `keywords`, `pull_requests_enabled`, and `pages_enabled`.

This PR fixes #2675

**Notes for Reviewers**

**Signed commits**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: augur/tasks/github/repo_info/core.py [hunk 1]
```diff
@@ -187,34 +271,40 @@ def repo_info_model(key_auth, repo_orm_obj, logger):
...
 
-    # Put all data together in format of the table
     logger.info(f'Inserting repo info for repo with id:{repo_orm_obj.repo_id}, owner:{owner}, name:{repo}\n')
     rep_inf = {
         'repo_id': repo_orm_obj.repo_id,
         'last_updated': data['updatedAt'] if 'updatedAt' in data else None,
         'issues_enabled': data['hasIssuesEnabled'] if 'hasIssuesEnabled' in data else None,
         'open_issues': data['issues']['totalCount'] if data['issues'] else None,
-        'pull_requests_enabled': None,
+        'pull_requests_enabled': 'true' if repo_data.get('has_projects') else 'false',
```
Comment (sgoggins) [inline augur/tasks/github/repo_info/core.py:290]: We are not using the `.get` syntax elsewhere. I am not sure I love the nesting, and I agree it is useful to get these data elements we previously ignored. 

Can you share a rationale for this design choice?

Comment (MoralCode) [reply augur/tasks/github/repo_info/core.py:290]: get syntax is better because it wont crash if the key doesnt exist (will return a default value, which is none by default)

I use it all the time

**Comment by MoralCode** [comment]
> I would like to see Unit tests written for these functions in a way that validates the existence of the end point and incorporates the use of edge cases. i.e. valid cases with an empty URL, invalid cases where there is not a 404 but its empty, etc.).

Unit tests for API dependent stuff is... a little trickier to do. especially if the function being tested is making the web calls on its own, not expecting the data to be passed into them.

**Comment by MoralCode** [comment]
This will also conflict with #3597. Im going to say that this should wait for that PR to merge, and then be adjusted to reuse the same function for fetching the correct domain name to use in Github API calls.

**Comment by MoralCode** [comment]
Hello! Just wanted to check in to see if you were still interested in helping the maintainers merge this PR. We noticed it has been a little while since this last had activity, and are considering closing it or taking it over if it remains in its current state.

Please react to or reply to this to confirm your interest in the next 7 days or let us know if you are no longer interested in this so we can best prioritize everyone's contributions.

Thanks!
